common/test/acceptance/pages/studio/overview.py

```diff
@@ -530,6 +530,13 @@ def change_problem_release_date_in_studio(self):
         self.q(css=".action-save").first.click()
         self.wait_for_ajax()
 
+    def select_additional_settings_tab(self):
+        """
+        Select the additional settings tab
+        """
+        self.q(css="#additional_settings").first.click()
+        self.wait_for_ajax()
```

Comment:
It's probably not an ajax call. We might have to wait for an element such as any of our radio buttons to appear. None type would be the most suitable since it's always visible.